injected/src/utils.js

```diff
@@ -712,6 +730,12 @@ export function computeEnabledFeatures(data, topLevelHostname, platformVersion,
                     return false;
                 }
             }
+            // Check that the platform supports maxSupportedVersion checks and that the feature has a maxSupportedVersion
+            if (feature.maxSupportedVersion && platformVersion) {
```

Comment:
Dunno if we should add this, it's not supported on other platforms and in the config syntax. I think I'll drop it.